### PR TITLE
WS-2219: fix events workspaceId non-nullable

### PR DESCRIPTION
**Pull request by Katsuya-Tomioka**

### CODE REVIEW

File: model/src/main/java/com/basistech/rosette/apimodel/EventsOptions.java [hunk 1]
```diff
@@ -20,6 +20,10 @@
 import lombok.Builder;
 import lombok.Value;
 
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
```
Comment (TomHubbardBT) [inline model/src/main/java/com/basistech/rosette/apimodel/EventsOptions.java]: Do you need notempty/notnull to get notblank? I know they functionally build off of each other.

File: model/src/main/java/com/basistech/rosette/apimodel/EventsOptions.java [hunk 2]
```diff
@@ -37,5 +41,6 @@
     /**
      * @return the modelType to use.
```
Comment (TomHubbardBT) [inline model/src/main/java/com/basistech/rosette/apimodel/EventsOptions.java]: whats the deal with modelType?

**Review by TomHubbardBT — COMMENTED** [review]
just those 2 comments about additional imports and the comment.

File: model/src/main/java/com/basistech/rosette/apimodel/EventsOptions.java (hunk 1 above)
Comment (Katsuya-Tomioka) [reply model/src/main/java/com/basistech/rosette/apimodel/EventsOptions.java]: relied too much on auto-import in intellij. Fixed.

File: model/src/main/java/com/basistech/rosette/apimodel/EventsOptions.java (hunk 2 above)
Comment (Katsuya-Tomioka) [reply model/src/main/java/com/basistech/rosette/apimodel/EventsOptions.java]: fixed.

**Review by TomHubbardBT — APPROVED** [review]
LGTM

**Pull request merged**